Paccy10/olympus-api
# Feature(users): update user profile

**What does this PR do?**

Adds update user profile functionality

**Description of Task to be completed?**

A registered and authenticated user can update their profile

**How should this be manually tested?**

- Checkout to the branch `ft-update-user-profile`
- Run the app with `flask run`
- Go to postman and send a `PUT` request to `/users/profile` with the required data
- The user profile should be updated

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

N/A

**Screenshots (if appropriate)**

N/A

**Questions:**

N/A
